### DISCUSSION TITLE
[feature request] quick filter the history list

### DISCUSSION THREAD

**Discussion by breaker27** [doublecmd/doublecmd#1949] [Suggestions]
I regularly filter the normal file/directory list by entering a part of the directory name to navigate.

I'd like to have this in the history as well.

**Use-case:** I want to go again to a folder where I was 10 minutes ago. I open the history with Ctrl-h. Then I want to start typing and the list should filter the directories in the history list so I can quickly select the path I want to go to.

That would really speed up navigating for me. :)

**Comment by Skif-off** [comment]
[Tree View Menu](https://doublecmd.github.io/doc/en/configuration.html#ConfigTreeMenu)?

**Comment by breaker27** [comment]
Wow! I didn't know that feature.
Yes, this solves my use-case. Thanks for the info.
